### ISSUE TITLE
No second page PDF suppprt

### ISSUE THREAD

**Issue by outlawjanitor** [r0adkll/DeckBox#25]
Hello, love your app, it's really great. Thanks for making it. 

However I've experienced one issue. Maybe it's my fault, so please tell me if I'm doing something wrong so I can correct it for the future. If not, I hope you can fix this in your app. 

When you do the export decklist feature, it gives you the very handy PDF which you can easily share and then print. This is great and I love this. However, there is only support for a 1 page PDF. My decklist was spilling over into a second page, but the given PDF was only one page long, so some of the cards on my decklist were cut off and not visible. 

As I'm typing this, I now realize that the error may only occur with the exact number of cards that I have in my decklist, which may explain why it's a small overlooked code error. If an email address is provided, I could send over the PDF for reference. You would see that only the very last card is cut off and not fully visible at the bottom. So maybe your app does allow for 2 page PDF's, but just not when it's this close to the cutoff point. Maybe it's a margins issue?

Anyway, hoping you can help me with this problem or provide any info. Again, thanks for the great app and thanks in advance for any help.

**Comment by r0adkll** [comment]
Hi! Thanks for the support and double thanks for opening an issue on GitHub as this will allow me to keep track of it better and to provide quicker feedback.

Yes, the application only supports single page PDF exporting as I imagine that it is preferable to only submit 1 page when entering tournaments. I tried to improve on this in the latest update by condensing energy and supporter cards into single lines where possible. 

If you post your PTCGO export here I can play around with the pdf exporter and see if I can find a better solution to potentially long lists. Or you can email me at veedubusc@gmail.com

**Comment by outlawjanitor** [comment]
Closed as in fixed?

**Comment by r0adkll** [comment]
Yes! I'm pushing out **v1.7.1** today that fixes this issue. I modified the exporter to take a two column approach if you have a greater number of non-duplicate cards in your deck.

**Comment by outlawjanitor** [comment]
Awesome, thanks dude!

On Sep 1, 2018 3:28 PM, "Drew Heavner" <notifications@github.com> wrote:

Yes! I'm pushing out *v1.7.1* today that fixes this issue. I modified the
exporter to take a two column approach if you have a greater number of
non-duplicate cards in your deck.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/r0adkll/DeckBox/issues/25#issuecomment-417885102>, or mute
the thread

**Comment by r0adkll** [comment]
No problem! The update is now live on the play store
